docs-sources/includes/_Events.md

```diff
@@ -1,8 +1,56 @@
 # Events
 
-We have seen in the [States](#states) section of this user guide that
-states can be used to emit events between players. Let's dig a bit deeper
-into how this can be used.
+We have seen in the [States](#states) section of this user guide that states can be used to emit events between players.
+Before explaining how to send events, we will first see how MAGE send events to the clients, and how to configure the server to do so.
+
+## How does it work ?
+
+Events sending and receipt is done via **Message Stream**.
+
+**Message Stream** is a protocol used by MAGE servers and its clients in order to communicate. This protocol is implemented through different transports, which are the following:
```

Comment:
I think we don't need to make **Message Stream** bold more than once - the first time its mentioned. Otherwise it feels a bit like we're screaming at people :)

Reply:
Ok, I didn't see this like that